src/java.base/share/classes/java/util/zip/ZipCoder.java

```diff
@@ -56,28 +56,27 @@ public static ZipCoder get(Charset charset) {
     }
 
     /**
-     * This enum represents the three possible return values for
+     * These values represents the three possible return values for
      * {@link #compare(String, byte[], int, int, boolean)} when
      * this method compares a lookup name to a string encoded in the
      * CEN byte array.
      */
-    enum Comparison {
-        /**
+    static final byte
+        /*
          * The lookup string is exactly equal
          * to the encoded string.
-          */
-        EXACT_MATCH,
-        /**
+         */
+        EXACT_MATCH = 1,
```

Comment:
Would there be any (positive) performance implications of starting these at zero?

Reply:
It saves a byte since there are bytecode constants for `0`-`2` but not for `3` :-) Fixed